qadence/models/quantum_model.py

```diff
@@ -165,9 +169,17 @@ def sample(
         endianness: Endianness = Endianness.BIG,
     ) -> list[Counter]:
         params = self.embedding_fn(self._params, values)
-        return self.backend.sample(
+        output = self.backend.sample(
             self._circuit, params, n_shots=n_shots, state=state, endianness=endianness
         )
+        if self._noise_model is None:
+            return output
+        # corrupt bitstrings with simple readout noise
+        elif self._noise_model.name == "ReadoutError":
```

Comment:
That's the point: readout errors aren't generated by noise. I think naming everything `error_model` makes it more generic.